www/about/faq.spt

```diff
@@ -55,6 +55,10 @@ title = _("Frequently Asked Questions")
             '</a>'|safe) }}</p>
     </dd>
 
+    {{ dt(_("Can Liberapay be used to laundry money?"), 'laundry-money') }}
```

Comment:
:D that's a funny way of phrasing it, but nobody would ever ask "is it okay if I use your service to do something illegal?" as the answer is obviously "no".